src/Trakx.CryptoCompare.ApiClient/Trakx.CryptoCompare.ApiClient.csproj

```diff
@@ -17,11 +17,12 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="5.0.0" />
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
     <PackageReference Include="Polly" Version="7.2.1" />
     <PackageReference Include="Serilog" Version="2.10.0" />
     <PackageReference Include="System.Reactive.Linq" Version="5.0.0" />
     <PackageReference Include="Trakx.Utils" Version="0.1.13" />
+    <PackageReference Include="Trakx.Websocket" Version="0.3.31-a4763ee" />
```

Comment:
In dotnet environment, websocket is usually written with "S" in upper case ("WebSocket"). not a big deal.